src/applications/lgy/coe/status/sass/coe-status.scss

```diff
@@ -1,2 +1,6 @@
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 @import "../../../../../platform/forms/sass/m-schemaform";
+
+.document-list {
+  list-style-type: none;
+}
```

Comment:
I _was_ going to change the document list from a `<div>` wrapped list into an actual `<ul>` with `<li>`s, but need a11y feedback first... and it's out of scope.